ui/src/context/AppContext.tsx

```diff
@@ -26,8 +26,8 @@ export type AppContextProps = {
 export const AppContext = createContext({} as AppContextProps);
 
 export const AppContextProvider: FunctionComponent<AppContextProviderProps> = ({ children }) => {
-  const [isGroupsLoading, setGroupsLoading] = useState<boolean>(false);
```

Comment:
the setter not being the proper syntax bothered me when I was in the area debugging the error message, so I fixed it.